authentication_handlers_test.go

```diff
@@ -417,6 +420,8 @@ func TestAuthenticationCreateBadRequestInvalidResourceType(t *testing.T) {
 	)
 	c.Request().Header.Add("Content-Type", "application/json;charset=utf-8")
 
+	c.Echo().Binder = echoBinder
+
```

Comment:
well I am not sure about this ... yes we need it here but I think we could set Binder() for all our tests on one place ... I started discussion on Slack ... maybe I am wrong but this can probably lead to separate PR where we set Binder() for all our tests (so in this PR we can leave it the way you did it) 🤔 😇